hasjob/models/domain.py

```diff
@@ -52,6 +52,10 @@ def use_title(self):
     def has_profile(self):
         return bool(self.title and self.description)
 
+    def post_with_logo(self):
+        """ Returns the first post with a company logo """
+        return self.jobposts.filter(JobPost.company_logo != None).first()
```

Comment:
And add a `# NOQA` comment at the end of PyFlakes will cry foul about the `!= None`.